example-projects/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-all.zip
```

Comment:
Why is this 7.6?

Reply:
Excellent question. Basicly, I want renovate to update this for me :) Thus the activation in the config.

Reply:
Haha, alright :)